A bunch of mobile formatting changes.

dburka opened the pull request:
Fixed About page on mobile. Icons are all larger. Added Remix icons to the header nav. Added a gradient background to the main box on homepage.

sgarrity commented:
Looks good.

The secondary header links (Request..., Github, About) is too wide for a 320 or 360px wide phone:

<img width="384" alt="image" src="https://user-images.githubusercontent.com/58062/120189452-c0251e00-c1ed-11eb-9aef-8939b3b32738.png">

I think the contrast is a bit low on the main headline ("Free, open...") and the line below ("Free for use in...").

I'd drop the hover effect on the three links (Request..., Github, About). The individual hover effect is good, but if I hover over "About" and see a change over on the "Request" link, it's distracting.

The speed of the "on" animations on the download button and the icon tiles feels slow. I'd do a fast-on (or instant on), slightly-slower-off.

sgarrity commented:
> The secondary header links (Request..., Github, About) is too wide for a 320 or 360px wide phone:

We could hide the icons and make the font a bit smaller at really small sizes (under 400px or so).

Could also reduce the `gap` value on `.topbarLinks` from `2rem` to `1rem`

dburka commented:
I addressed all of those issues and made the box on the homepage better as per suggestions in Slack. And, made the pointer a hand when you hover over the icons.
